pom.xml

```diff
@@ -213,6 +213,47 @@
                     <artifactId>git-commit-id-plugin</artifactId>
                     <version>${git-commit-id-plugin.version}</version>
                 </plugin>
+
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-antrun-plugin</artifactId>
+                    <version>1.8</version>
+                    <executions>
+                        <execution>
+                            <id>ktlint</id>
+                            <phase>validate</phase>
+                            <configuration>
+                                <target name="ktlint">
+                                    <java taskname="ktlint" dir="${basedir}" fork="true" failonerror="true"
+                                          classpathref="maven.plugin.classpath" classname="com.pinterest.ktlint.Main">
+                                        <arg value="src/**/*.kt"/>
+                                    </java>
```

Comment:
Is this going to lint the test sources as well? It'd be great if we had linting set up for test sources too (even though we don't really have them yet 😄)

Reply:
If they will be located at **src/test/kotlin/...**, then they are linted as well. Just tested them by adding a test/kotlin directory into the src of a module.

Reply:
Ah, I forgot how Maven projects are structured 😄 (been working with TypeScript lately where tests have their own top-level directory). Nice, thanks!